mitmproxy/net/dns/https_record.py

```diff
@@ -0,0 +1,91 @@
+import base64
+import ipaddress
+import struct
+
+
+def unpack_params(data: bytes, offset: int) -> dict:
```

Comment:
Nit: You can preface this method with `_` to mark it as private. We certainly don't do that consistently across the codebase, but we should do it for new code :)